Sources/NibLoadable.swift

```diff
@@ -14,7 +14,7 @@ public extension NibLoadable {
   /// By default, use the nib which have the same name as the name of the class,
   /// and located in the main bundle
```

Comment:
Also we should update the comment (which serves for the enervated documentation) here. Not in main bundle anymore.